src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -871,16 +936,21 @@ public String constructUserData(boolean graphAlreadyBuilt) {
         manifest.s3UploadPath = getS3FolderURI().toString();
         manifest.statusFileLocation = String.format("%s/%s", webDir, OTP_RUNNER_STATUS_FILE);
         manifest.uploadOtpRunnerLogs = true;
+        String otpRunnerManifestS3Filename;
```

Comment:
This needs a comment describing that the filename will change depending on the context and why this is the case.